.circleci/config.yml

```diff
@@ -127,6 +127,33 @@ commands:
           name: "Running tests matching '<< parameters.pattern >>'"
           command: "riot -v run -s '<< parameters.pattern >>'"
 
+  run_tox_scenario_with_testagent:
+    description: Run scripts/run-tox-scenario with setup, caching persistence and the testagent
+    parameters:
+      pattern:
+        type: string
+      wait:
+        type: string
+        default: ""
+    steps:
+      - checkout
+      - restore_tox_cache
+      - when:
+          condition:
+            << parameters.wait >>
+          steps:
+            - run:
+                name: "Waiting for << parameters.wait >>"
+                command: tox -e 'wait' << parameters.wait >>
+      - run: SNAPSHOT_CI=1 docker-compose up -d memcached redis testagent
+      - run:
+          command: docker-compose logs -f
+          background: true
+      - run:
+          name: "Run scripts/run-tox-scenario"
+          command: DD_TRACE_AGENT_URL=http://localhost:8127 ./scripts/ddtest scripts/run-tox-scenario '<< parameters.pattern >>'
```

Comment:
lets use something other than `8127`, since we seem to do incrementing ports for our products (`8125` for Dogstatsd, `8126` for Trace agent), maybe `9126`?

Reply:
(not a strong opinion, just a suggestion)

Reply:
Done